doc/sphinxman/source/build_planning.rst

```diff
@@ -1300,6 +1309,23 @@ a few other factors. Either way, this file should contain CMake's testing
 output, as well as everything that was printed to the screen.
 
 
+.. _`faq:pytest`:
+
+How to test a Psi4 installation
+-------------------------------
+
+``ctest`` requires a connection to source files and ``cmake``
+machinery and so can only be performed from :samp:`{objdir}`
+(staged installation). To test an installed |PSIfour| (full or staged
+installation), a limited number of "smoke" tests are available to be
+run via pytest. ::
+
+    python -c "import psi4; psi4.test()"
+
+For the above to work, |PSIfour| must be detectable as a Python
+module. See the note at :ref:`faq:runordinarymodule` for setup if needed.
+
+
```

Comment:
Can't add to `ctest`. But can make a `cmake` target of it so can `make pytest` and that'll run it. List is getting extensive.

* from `{top-level-psi4-dir}/tests/pytest`, set PYTHONPATH, `python -m pytest -v`
* from anywhere, set PATH, `psi4 --test`
* from anywhere, set PYTHONPATH, `python -c "import psi4; psi4.test()"`
* from `{objdir}`, `make pytest`